examples/server-side-rendering/package.json

```diff
@@ -55,12 +57,34 @@
     "@angular/cli": "7.3.2",
     "@angular/compiler-cli": "7.2.5",
     "@angular/language-service": "7.2.5",
+    "@types/jest": "24.0.6",
     "@types/node": "11.9.4",
     "cpy-cli": "2.0.0",
     "express": "4.16.4",
     "http-server": "0.11.1",
+    "jest": "24.1.0",
+    "jsdom": "13.2.0",
+    "node-fetch": "2.3.0",
     "reflect-metadata": "0.1.9",
+    "start-server-and-test": "1.7.11",
+    "ts-jest": "24.0.0",
     "ts-loader": "5.3.3",
     "typescript": "3.2.4"
+  },
+  "jest": {
+    "setupFiles": [],
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "transformIgnorePatterns": [
+      "<rootDir>/lib/polyfills/*.js"
+    ],
+    "testRegex": "(/lib/__tests__/.*|(\\.|/)(test|spec))\\.(js?|ts?)$",
+    "moduleFileExtensions": [
+      "ts",
+      "js",
+      "json",
+      "node"
```

Comment:
why is `node` an extension? for jsdom?

Reply:
I'm only using jsdom to parse the HTML into DOM objects easier to tests.
I don't actually need a browser context.

Reply:
sorry I didn't read you comment correctly. You're right.